hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/IncrementalTableBackupClient.java

```diff
@@ -437,6 +445,29 @@ protected void walToHFiles(List<String> dirPaths, List<String> tableList) throws
     }
   }
 
+  private void incrementalCopyBulkloadHFiles(FileSystem tgtFs, TableName tn) throws IOException {
+    Path bulkOutDir = getBulkOutputDirForTable(tn);
+    FileSystem fs = FileSystem.get(conf);
+
+    if (fs.exists(bulkOutDir)) {
```

Comment:
When does the output dir not exist? Is that an error case?

Reply:
the bulk output dir doesn't exist if there were no bulkloaded files that we need to re-split. I agree that the check doesn't do a good job of informing you as to why the output directory might exist. I'm happy to do the following

1. Add a boolean flag to track if there are files that we've bulkloaded + re-split
2. Add a comment here to explain why this check is necessary

Reply:
Definitely, it's how I noticed this. The first portion of the new test doesn't do any bulkloads, so was running into issues here until I added this conditional